sdk/python/assets/data/data.ipynb

```diff
@@ -221,6 +221,33 @@
     "returned_job.studio_url"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Authenticate with user identity\r\n",
+    "\r\n",
+    "When running a job on a compute cluster, you can also use your user identity to access data. To enable job to access data on behald of you, specify **identity= AzureMLOnBehalfOfCredential()** in job definition, as shown below. For more details, see [Accessing storage services](https://learn.microsoft.com/azure/machine-learning/how-to-identity-based-service-authentication)"
```

Comment:
AzureMLOnBehalfOfCredential here should be UserIdentity

Reply:
Fixed